packages/protocol/src/message-kinds/rfq.ts

```diff
@@ -77,9 +74,9 @@ export class Rfq extends Message<'rfq'> {
    * @throws if rfq's claims do not fulfill the offering's requirements
    */
   async verifyClaims(offering: Offering | ResourceModel<'offering'>) {
-    const { areRequiredCredentialsPresent } = pex.evaluateCredentials(offering.data.requiredClaims, this.claims)
+    const requiredCredentialsPresent = PresentationExchange.selectCredentials(this.claims, offering.data.requiredClaims)
```

Comment:
This is a nit-pick, but variable name is slightly misleading IMO, could do either one of these...

```suggestion
const requiredCredentialsPresent = PresentationExchange.selectCredentials(this.claims, offering.data.requiredClaims).length > 0
```

And then update the `if ()` condition below like this

```typescript
    if (!requiredCredentialsPresent) {
      throw new Error(`claims do not fulfill the offering's requirements`)
    }
```

Or, could do...

```suggestion
    const credentials = PresentationExchange.selectCredentials(this.claims, offering.data.requiredClaims)
```

and then keep the `if ()` condition as-is (I think I prefer this way)